apps/angular-demo/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
```

Comment:
most of the stuff in the angular-demo is just boilerplate and oyu can skip it. i didnt write it